gccg_transport_api.h

```diff
@@ -265,20 +275,18 @@ GCCG_INTERFACE GccgReturnStatus GccgConnectionDestroy(GccgConnectionHandle handl
  * @return A value from the GCCG_INTERFACE enumeration.
  */
 GCCG_INTERFACE GccgReturnStatus GccgTxPayload(GccgConnectionHandle handle,
-                                              const char *payload_json_str,
-                                              GccgMediaElements media_array,
+                                              const GccgBuffer *buffer,
                                               void* user_cb_param_ptr,
                                               int timeout_microsecs);
 
 /**
  * Free an array of receive buffers that was used by the GccgRxCallback() callback API function. This API is thread-safe.
  *
- * @param media_array Pointer to a structure that contains an array of media elements to free and the number of elements
- *                    in the array.
+ * @param buffer Pointer to a GccgBuffer*buffer that is to be freed
  *
  * @return A value from the GCCG_INTERFACE enumeration.
  */
-GCCG_INTERFACE GccgReturnStatus GccgRxFreeBuffer(GccgMediaElements *media_array);
+GCCG_INTERFACE GccgReturnStatus GccgRxFreeBuffer(const GccgRxCbData *buffer);
```

Comment:
The parameter type here should be `GccgBuffer`, not `GccgRxCbData`. Please also, update the typo in the comment above on line 285 (see `GccgBuffer*buffer`).